x-pack/plugins/security_solution/public/management/components/artifact_entry_card/artifact_entry_card.tsx

```diff
@@ -37,6 +37,8 @@ export interface CommonArtifactEntryCardProps extends CommonProps {
    */
   policies?: MenuItemPropsByPolicyId;
   loadingPoliciesList?: boolean;
+  // Artifact specific decoration to be displayed in the card section
+  Decorator?: React.NamedExoticComponent<ArtifactEntryCardDecoratorProps>;
```

Comment:
Interesting that you chose to make the type explicitly a function component. Why not just `React.ComponentType<>`?

Reply:
yeah, no reason 😁 
e22e8acabfaca5bed15621b0a443022c84323136

Reply:
yeah, no reason 😁
e22e8acabfaca5bed15621b0a443022c84323136